src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -987,19 +987,25 @@ void LedgerConsensusImp::accept (std::shared_ptr<SHAMap> set)
     WriteLog (lsDEBUG, LedgerConsensus)
         << "Applying consensus set transactions to the"
         << " last closed ledger";
+
     {
-        MetaView accum(*newLCL, tapNONE);
+        OpenView accum(&*newLCL);
         assert(accum.closed());
         applyTransactions (set.get(), accum,
-            newLCL, retriableTransactions);
-        accum.apply(*newLCL,
-            deprecatedLogs().journal("LedgerConsensus"));
+            newLCL, retriableTransactions, tapNONE);
+        accum.apply(*newLCL);
     }
 
     // retriableTransactions will include any transactions that
     // made it into the consensus set but failed during application
     // to the ledger.
 
+    // Make a copy for OpenLedger
```

Comment:
Yowza.  What's this?  Did I miss this?  Where does this get defined, is it off by default?  Why do we need it?